heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -322,10 +323,11 @@ public QueryTrace buildTrace() {
 
                         fetchSeries.addAnnotation(series.toString());
                         fetches.add(() -> metricBackend.fetch(
-                            new FetchData.Request(source, series, range, options),
+                            new FetchData.Request(series, range, options),
```

Comment:
Why remove metric_type  in the request ? I think this is the root cause of the IT tests failure. 
Make sure every fetchData request is not defaulting to metric_type=Point